addons/mail/models/ir_attachment.py

```diff
@@ -1,12 +1,14 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, models
+from odoo import models, fields
 from odoo.exceptions import AccessError
 
 class IrAttachment(models.Model):
     _inherit = 'ir.attachment'
 
+    link_preview_ids = fields.One2many('mail.link.preview', 'attachment_id', string='link preview data')
```

Comment:
why do we need to have multiple preview for the same attachment?

Reply:
I don't understand this comment there is no `one2one` relation.
And I don't want to add a field on the `ir.attachment` table.

Do we have a better way to handle this ?